# Generated for NaoQi method ALSentinel.exitNaoqi
# exit naoqi (warning: it will really exit Naoqi)

---


